### DISCUSSION TITLE
SQL Code not colour coded

### DISCUSSION THREAD

**Discussion by peterw-github** [community/community#48539] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hey guys!

<br>
<br>

I was just wondering, in my other SQL files in the same repo, the code is colour coded:

![image](https://user-images.githubusercontent.com/108731810/221320537-6e222966-93d8-4ede-9a14-8d9369a47eaf.png)

<br>
<br>

But in one particular SQL file, the code isn't colour coded, it's just all white:

![image](https://user-images.githubusercontent.com/108731810/221320601-45b907f0-657b-4708-b627-f9be588e1942.png)

<br>
<br>

I'm quite stumped unfortunately. Any help/info as to what's happened would be appreciated.

**Comment by pop-cm** [comment, marked as answer]
I don't have the info as to why, but I found the source of the error to be the ".NEXTVAL" inclusions. Upon removing those, the syntax colouring works.

**Reply by peterw-github** [reply]
Ahhh, thanks for that pop-cm, I definitely appreciate the help (: nice catch as well, I definitely wouldn't of guessed the .NEXTVAL inclusions.
